src/main/java/com/hedvig/contract/apartments/ApartmentAgreement.kt

```diff
@@ -111,5 +112,42 @@ data class ApartmentAgreement(
         isInception = compatAgreement.isInception ?: false
       )
     }
+
+    fun fromApartmentContractDto(dto: ApartmentManualAgreementRequest): ApartmentAgreement {
+      return ApartmentAgreement(
+        id = dto.id,
+        basePremium = dto.basePremium,
+        addOnPremium = dto.addOnPremium,
+        lineOfBusiness = dto.lineOfBusiness,
+        from = dto.from,
+        to = dto.to,
+        address = Address(
+          dto.addressStreet,
+          dto.addressCoLine,
+          dto.addressPostalCode,
+          dto.addressCity,
+          dto.addressCountry
+        ),
+        coInsured = if (dto.numberCoInsured - 1 > 0) {
```

Comment:
Does the numberCoInsured include the one person insured? Shouldn't it be 0 or null if there are no consinured?

```suggestion
        coInsured = if (dto.numberCoInsured - 1 > 0) {
```